src/application.js

```diff
@@ -40,6 +40,7 @@ export const BellaApplication = GObject.registerClass(
       });
       this.add_action(quitAction);
       this.set_accels_for_action("app.quit", ["<primary>q"]);
+      this.set_accels_for_action("app.preferences", ["<primary>comma"]);
```

Comment:
Any particular reason why you went for this?

Reply:
Standard Keyboard Shortcuts

Preferences Ctrl+, Opens the preferences window

https://developer.gnome.org/hig/reference/keyboard.html